### DISCUSSION TITLE
Extensions information

### DISCUSSION THREAD

**Discussion by ioisgg** [standardnotes/forum#2560] [General] [answered]
Are all the Extensions listed on https://standardnotes.org/extensions maintained by Standard Notes Ltd. ?

Thanks!
G

**Comment by JaspalSuri** [comment, marked as answer]
Hi @gg-user,

The extensions offered through the [Extended plans](https://standardnotes.org/extensions) are maintained by Standard Notes and the original maintainers of derived extensions. 

Each editor and subsequent updates to them are vetted by the dev team to ensure that they are safe to use before they reach users. The ways in which the apps communicate with extensions has been audited by a third-party as well. 🙂

For more information, please [see here](https://standardnotes.org/help/77).
